Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -444,10 +445,10 @@ void TextureMapperLayer::collectDamageSelf(TextureMapperPaintOptions& options, D
 {
     ASSERT(m_damagePropagation);
 
+    m_previousLayerRectInGlobalCoordinateSpace = { };
```

Comment:
```suggestion
    m_previousLayerRectInGlobalCoordinateSpace.reset();
```

this is a personal preference, but calling `reset()` feels more explicit to me. It makes it clear that this is an optional. Otherwise I might expect that this is the same as the other variables below